saas/justfile

```
@@ -12,6 +12,14 @@ aws_local_env := if relative_path =~ '^/infra/dev.*' {
        "NO_ENV_DEFINED"
     }
 
+aws_local_region := if relative_path =~ '^/infra/.*/eu-west-3.*' {
+        "eu-west-3"
+    } else if relative_path =~ '^/infra/.*/eu-center-1' {
```

Not sure but the other paths have this
```suggestion
    } else if relative_path =~ '^/infra/.*/eu-center-1.*' {
```